tests/foreman/cli/test_organization.py

```diff
@@ -347,31 +211,26 @@ def test_positive_add_subnet_by_name(self):
 
     @run_only_on('sat')
     @tier2
-    @stubbed()
     def test_positive_add_subnet_by_id(self):
-        """@test: Add a subnet by its ID
+        """Add a subnet to organization by its ID
 
         @feature: Organization
 
         @assert: Subnet is added to the org
         """
-        for name in (gen_string('alpha'), gen_string('numeric'),
-                     gen_string('alphanumeric'), gen_string('utf8'),
-                     gen_string('latin1')):
-            with self.subTest(name):
-                org = make_org()
-                new_subnet = make_subnet({'name': name})
-                Org.add_subnet({
-                    'name': org['name'],
-                    'subnet-id': new_subnet['id'],
-                })
-                org = Org.info({'id': org['id']})
-                self.assertIn(name, org['subnets'][0])
+        org = make_org()
+        new_subnet = make_subnet()
+        Org.add_subnet({
+            'name': org['name'],
+            'subnet-id': new_subnet['id'],
+        })
+        org = Org.info({'id': org['id']})
+        self.assertIn(new_subnet['name'], org['subnets'][0])
 
     @run_only_on('sat')
     @tier2
     def test_positive_remove_subnet_by_name(self):
-        """@Test: Add a subnet and then remove it by its name
+        """Add a subnet to organization and then remove it by its name
```

Comment:
That's what @oshtaier suggested few comments above :smile: For me both options are ok